store/tikv/gcworker/gc_worker.go

```diff
@@ -1302,6 +1309,12 @@ func RunGCJob(ctx context.Context, s tikv.Storage, safePoint uint64, identifier
 	if concurrency <= 0 {
 		return errors.Errorf("[gc worker] gc concurrency should greater than 0, current concurrency: %v", concurrency)
 	}
+
+	err = gcWorker.saveSafePoint(gcWorker.store.GetSafePointKV(), tikv.GcSavedSafePoint, safePoint)
+	if err != nil {
```

Comment:
please also add this to RunDistributedGCJob.

Reply:
done.